src/components/Chat/Chat.module.css

```diff
@@ -0,0 +1,58 @@
+.chatRoom {
+    font-size: 1em;
```

Comment:
I think `font-size: 1em` doesn't do anything, it just says make it the current size